.dev/docker/db/docker-entrypoint-initdb.d/setup.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE gcon_development;
```

Comment:
That's the name of the DB that the app is configured to use when run in dev mode right now. I don't know how we're getting this deployed to production yet exactly so I didn't want to spin my wheels adding configuration stuff that might need to be scrapped later.